### PR TITLE
FINERACT-2114:  fix EMI calc get payment details

### PR DESCRIPTION
**Pull request by janez89**

### CODE REVIEW

**Review by adamsaghy — APPROVED** [review]
LGTM

File: fineract-progressive-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/ProgressiveLoanScheduleGenerator.java [hunk 1]
```diff
@@ -103,11 +104,15 @@ public LoanScheduleModel generate(final MathContext mc, final LoanApplicationTer
 
         prepareDisbursementsOnLoanApplicationTerms(loanApplicationTerms);
 
+        final ArrayList<DisbursementData> disbursementDataList = new ArrayList<>(loanApplicationTerms.getDisbursementDatas());
+        disbursementDataList.sort(Comparator.comparing(DisbursementData::disbursementDate));
```
Comment (galovics) [inline fineract-progressive-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/ProgressiveLoanScheduleGenerator.java]: I'm having the same question here as for PR https://github.com/apache/fineract/pull/4048

Comment (janez89) [reply fineract-progressive-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/ProgressiveLoanScheduleGenerator.java]: This branch was prepared to check some test cases on another feature branch.
Now rebased and up to date. I removed the unnecessary commits from this branch.

**Review by galovics — APPROVED** [review]

**Review by somasorosdpc — APPROVED** [review]
LGTM

**Pull request merged**